canonical-web-and-design/canonical.com
# Code cleanup

## Done

- Update README to include info on API key
- Add Prettier as a dep
- Run Prettier over whole site
- Fix invalid HTML

## QA

Note: Due to Prettier, it'll be easier to review code by following commits

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Sanity check code
- Verify all tests and checks pass
